fsm/leave_parent_ds_synced.go

```diff
@@ -17,7 +17,7 @@ var FsmLeaveParentDsSynced = music.FSMTransition{
 
 	PreCondition:  LeaveParentDsSyncedPreCondition,
 	Action:        LeaveParentDsSyncedAction,
-	PostCondition: func(z *music.Zone) bool { return true }, // XXX: TODO
+	PostCondition: LeaveVerifyCDSRemoval,
```

Comment:
Ok, so this one is called LeaveVerifyCDS*Removal*, but the other one is called LeaveCDSVerify (no "Add" anywhere). I'd like to have the tokens "Verify" and "CDS" in the same order everywhere and I'd like to have symmetry in naming between Add and Remove functions.

Reply:
Good point.